integration-test.ps1

```diff
@@ -0,0 +1,61 @@
+## ---------------------------------------------------
+##   Complete end to end integration test of the SDK
+## ---------------------------------------------------
+
+## How to configure this script
+## ----------------------------
+
+## 1. Select a certificate to use and put its thumbprint into $thumbprint:
```

Comment:
you could also consider putting these as parameters to the script, then it would prompt the user for values?

Reply:
Not sure - wouldn't the prompt show up before the script does anything? There's be no context around what was being asked for.